cloudferrylib/os/compute/nova_compute.py

```diff
@@ -14,11 +14,13 @@
 
 
 import copy
+from operator import attrgetter
 import time
 from sqlalchemy import exc
 
 from novaclient.v1_1 import client as nova_client
 from novaclient import exceptions as nova_exc
+import cfglib
```

Comment:
It is related to CloudFerry modules, so it is should be in the next imports block (below)